src/styles/testimonials.css

```diff
@@ -0,0 +1,104 @@
+.carousel-root {
+  width: 64%;
```

Comment:
Using percentages for layout widths can lead to unpredictable behavior on different devices. Consider using flexbox or grid for more consistent responsiveness. Let me know if you need help!